docker-compose/cassandra-4.0/README.md

```diff
@@ -0,0 +1,24 @@
+This directory provides two ways to start Stargate with Cassandra 4.0 using docker-compose.
+
+# Stargate with 3-node Cassandra 4.0 cluster
+
+Use the shell script `start_cass_40.sh`, which performs an orderly startup of containers 
+specified in `docker-compose.yml`:
+
+`./start_cass_40.sh`
+
+# Stargate with embedded Cassandra 4.0 (developer mode) 
+
+This alternate configuration that runs the Stargate coordinator node in developer mode, so that no
+separate Cassandra cluster is required. This can be run directly with the command:
+
+`docker-compose -f docker-compose-dev-mode.yaml up`
```

Comment:
Ahh. I think this answers my earlier question; there's default name, but one that can be overridden.